src/test/java/org/codehaus/mojo/properties/PropertyResolverTest.java

```diff
@@ -81,6 +81,19 @@ public void multipleValuesAreResolved()
     }
 
     @Test
+    public void propertyIncludesAnotherPropertyMoreThanOnce()

```

Comment:
To match the previous method, wouldn't `multipleOccurrencesOfTheSameValueAreResolved ` be more appropriate?